on-boot-script/README.md

```diff
@@ -49,6 +49,7 @@
     Examples:
     * Start a DNS Container [10-dns.sh](../dns-common/on_boot.d/10-dns.sh)
     * Start wpa_supplicant [on_boot.d/10-wpa_supplicant.sh](examples/udm-files/on_boot.d/10-wpa_supplicant.sh)
+    * Add a persistent ssh key [on_boot.d/15-add-ssh-key.sh](examples/udm-files/on_boot.d/15-add-ssh-key.sh)
```

Comment:
might be helpful to add that this is for `root`

Reply:
Updated PR to address this concern.